src/main/java/org/codehaus/plexus/archiver/AbstractUnArchiver.java

```diff
@@ -125,6 +128,16 @@ public void setOverwrite( final boolean b )
         overwrite = b;
     }
 
+    @Override
+    public FileMapper[] getFileMappers(  ) {
+        return fileMappers;
+    }
+
+    @Override
+    public void setFileMappers( final FileMapper[] fileMappers ) {
```

Comment:
The open bracket should be on new line

Reply:
I will fix this and squash tomorrow. Shall I just manually do that, or is it ok if I just apply the original Maven Eclipse Formatter settings? This will fix some more style bugs that already existed before.

Reply:
Both are ok. If you see formatting violations and want to fix them you are more than welcome to do so. But if you modify lines outside your changes please squash the commits and add second commit with only the formatting changes. You can append it to this PR if you modify a small number of files (if you want to reformat the whole project it is ok but would be best left to separate PR as it would take time to review).

Reply:
Understood. So I will go with the manual fix for now as it is more quick. But good to know for later.

Reply:
Fixed by https://github.com/codehaus-plexus/plexus-archiver/pull/100/commits/deacf5e4cb6e6a6e197b0887be687c99b1f6f96a.